torchgen/model.py

```diff
@@ -1616,6 +1617,9 @@ def strip_ret_annotation(r: Return) -> Return:
         if strip_view_copy_name and base_name.endswith("_copy"):
             base_name = base_name.replace("_copy", "")
 
+        if swap_scatter_inverse:
```

Comment:
nit: I think that going from view_scatter-op to view_inverse-op is morally the same as going from view_copy to view op. So I don't think you need a new flag: you can just do:
```
        if strip_view_copy_name:
                if base_name.endswith("_copy"):
                    base_name = base_name.replace("_copy", "")
                elif base_name.endswith("_scatter")
                    base_name = base_name.replace("_scatter, "_inverse")
```

Reply:
nice, makes it cleaner :)